CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 - The HTTP header parsing is now case insensitive.
 - The `put_api_requests` and `patch_api_requests` metrics for net devices were
   un-swapped.
+- Added `--version` flag to both Firecracker and Jailer.
```

Comment:
We cannot release a patch version from this commit because the master branch is not stable just now and includes other changes as well. Can you move this under the `# Unreleased` section? For creating a patch release we need a new branch that starts from the commit of the v0.21.0 release, and only has this commit on top of it. I can create a branch and then we can open a new PR against that release branch.

Reply:
okay. thanks for the explanation~ updated. ;)

Reply:
okay. thanks for the explanation. ;). updated.